### PR TITLE
Send text message to queue

### PR DESCRIPTION
**Pull request by raskhodchikov**
Testing sending message to  queue

### CODE REVIEW

File: src/client/TdlClient.cs [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
+
+        public void Request(Dictionary<string, object> jolokiaPayload)
+        {
+            var client = new RestClient($"http://{Host}:{Port}/api/jolokia");
+
+            var request = new RestRequest(Method.POST);
+            request.AddHeader("Content-Type", "application/json");
+            request.AddJsonBody(jolokiaPayload);
+
+            var response = client.Execute(request);
```
Comment (julianghionoiu) [inline src/client/TdlClient.cs]: This is good progress. I am pleased to see that you managed to publish a message to the Broker.
One thing to bare in mind, though. The Jolokia requests are using the Admin interfacge and should not be part of the production code. These type of requests are only part of the test code.

Comment (julianghionoiu) [reply src/client/TdlClient.cs]: This should be part of a support class that will implement the `test/specs/ClientSteps.cs:I receive the following requests`

**Comment by raskhodchikov** [comment]
Implemented the steps that are counting requests and responses.

File: test/specs/ClientSteps.cs [hunk 2]
```diff
@@ -1,74 +1,98 @@
...
-		public void ThenIShouldGetNoException()
-		{
-			ScenarioContext.Current.Pending();
-		}
-	}
+        [Then(@"the client should not publish any response")]
+        public void ThenTheClientShouldNotPublishAnyResponse()
+        {
+            Assert.AreEqual(0, requestQueue.GetSize(),
+                "The response queue has different size. Messages have been published.");
```
Comment (julianghionoiu) [inline test/specs/ClientSteps.cs]: The response queue is a different queue. The clients are reading requests from the request queue, applying a function and publishing the response to the response queue.

File: test/specs/ClientSteps.cs [hunk 3]
```diff
@@ -1,74 +1,103 @@
...
+            requestQueue = TestBroker.Instance.AddQueue($"{UniqueId}.req");
+            requestQueue.Purge();
 
-		[Then( @"the client should consume all requests" )]
-		public void ThenTheClientShouldConsumeAllRequests()
-		{
-			ScenarioContext.Current.Pending();
-		}
+            responseQueue = TestBroker.Instance.AddQueue($"{UniqueId}.resp");
+            responseQueue.Purge();
```
Comment (julianghionoiu) [inline test/specs/ClientSteps.cs]: The initialisation should be part of "I start with a clean broker".
Not a blocker.

**Pull request merged**